extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/deployAzureSQLDBWizard.ts

```diff
@@ -172,4 +189,33 @@ export class DeployAzureSQLDBWizard extends WizardBase<DeployAzureSQLDBWizard, W
 			level: azdata.window.MessageLevel.Error
 		};
 	}
+
+	public async refreshPages() {
+
+		const pageCount = this.wizardObject.pages.length;
+		for (let i = 1; i < pageCount; i++) {
```

Comment:
I assume starting at 1 because you're purposely skipping the EULA page? Why? This is where comments are very much appreciated since it isn't obvious from the code why you'd be starting here.